connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -123,6 +125,10 @@
     private static final String CONSUMER_POLL_TIMEOUT_MILLIS_DOC = "Timeout when polling source cluster.";
     public static final long CONSUMER_POLL_TIMEOUT_MILLIS_DEFAULT = 1000L;
 
+    public static final String CONSUMER_AUTO_OFFSET_RESET = "consumer.auto.offset.reset";
+    private static final String CONSUMER_AUTO_OFFSET_RESET_DOC = "Consumer Auto offset reset, defaults to earliest unless specify.";
```

Comment:
can leave off "unless specified" -- is redundant with "default".

Reply:
Updated "Consumer Auto offset reset, default to earliest unless specified."